packages/webapp/src/stories/Button/Button.stories.jsx

```diff
@@ -66,13 +75,38 @@ DisabledSmall.args = {
   sm: true,
 };
 
-const style = {
-  background: 'linear-gradient(to right, orange , yellow, green, cyan, blue, violet)',
+export const WithIconSmall = Template.bind({});
+WithIconSmall.args = {
+  color: 'primary',
+  sm: true,
+  children: (
+    <>
+      <div>Edit</div>
```

Comment:
It's a story so it's fine, but `div` should be `span` 😄

Reply:
I actually thought the flexbox `gap` would only apply to a block-level element!! I'm kind of surprised to learn it works on span!!